src/test/java/com/puppycrawl/tools/checkstyle/api/AbstractCheckTest.java

```diff
@@ -135,8 +136,9 @@ public void testTokenNames() {
         };
 
         check.setTokens("IDENT, EXPR, ELIST");
-        assertArrayEquals(new String[] {"IDENT, EXPR, ELIST"},
-            check.getTokenNames().toArray(), "unexpected result");
+        assertWithMessage("unexpected result")
+                .that(check.getTokenNames().toArray())
```

Comment:
We don't need `toArray` here
It can be `.that(check.getTokenNames().containsExactlyElementsIn(,,)`

Reply:
Sorry, missed that. Done.